### PR TITLE
[Cloud Tasks] Add HTTP Push Queue Sample

### PR DESCRIPTION
**Pull request by averikitsch**
Test projects will have to be granted permission for HTTP targets or wait until whitelisting is off.

### CODE REVIEW

File: cloudtasks/api/HttpTasksSample/Samples.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using System;
+
+namespace GoogleCloudSamples
+{
+    public partial class Samples
+    {
+        // [START cloud_tasks_create_http_task]
+        public static object CreateTask(
```
Comment (SurferJeffAtGoogle) [inline cloudtasks/api/HttpTasksSample/Samples.cs]: According to the new rubric, the START and END tags should enclose the using statements at the top and the complete class.

File: cloudtasks/api/HttpTasksSample/Samples.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            string projectId,
+            string location,
+            string queue,
```
Comment (SurferJeffAtGoogle) [inline cloudtasks/api/HttpTasksSample/Samples.cs]: According to the new rubric, each param should have a default value that looks like a real value the caller might pass in.  i.e.:
```
 string projectId = "YOUR-PROJECT-ID",
 string location = "global",
 string queue = "YOUR-QUEUE-ID",
```

File: cloudtasks/api/HttpTasksSample/Program.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using CommandLine;
+using System;
+
+namespace GoogleCloudSamples
```
Comment (SurferJeffAtGoogle) [inline cloudtasks/api/HttpTasksSample/Program.cs]: According to the new rubric, a command-line application is no longer necessary.  You can call the function directly from the test.

**Comment by SurferJeffAtGoogle** [comment]
The tests failed.

**Comment by averikitsch** [comment]
Looks like I rebased incorrectly. Will close and open cleaner PR.

**Pull request closed without merging**